hosted_control_planes/hcp-disconnected/hcp-deploy-dc.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-deploy-dc"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {hcp} in a disconnected environment
+:context: hcp-deploy-dc
+
+toc::[]
+
+In the context of hosted control planes, a disconnected environment is an {product-title} deployment that is not connected to the internet and that uses hosted control planes as a base. You can deploy {hcp} in a disconnected environment on bare metal or {VirtProductName}.
+
+{hcp-capital} in disconnected environments function differently than in standalone {product-title}:
+
+* The control plane is in the management cluster. The control plane is where the pods of the hosted control plane are run and managed by the Control Plane Operator. 
+* The data plane is in the workers of the hosted cluster. The data plane is where the workloads and other pods run, all managed by the HostedClusterConfig Operator.
+
+Depending on where the pods are running, they are affected by the `ImageDigestMirrorSet` (IDMS) or `ImageContentSourcePolicy` (ICSP) that is created in the management cluster or by the `ImageContentSource` that is set in the `spec` field of the manifest for the hosted cluster. The `spec` field is translated into an IDMS object on the hosted cluster.
+
+You can deploy {hcp} in a disconnected environment on the following networks: IPv4, IPv6, and dual stack. IPv4 is one of the simplest network configurations to deploy hosted control planes in a disconnected environment. IPv4 ranges require fewer external components than IPv6 or dual-stack setups.
```

Comment:
AFAIK OpenShift Virtualization (even outside HCP) currently supports only IPv4 and dual-stack but not IPv6 only that is going to be tech-preview starting with v4.18.
See: https://issues.redhat.com/browse/CNV-14598
I'd suggest to mention it.

Reply:
I will mention that. Thanks!